assets/core/route-controller-separate-sa-rolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+# needed to support the "use separate service accounts" feature.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  namespace: openshift-infra
+  name: system:openshift:sa-creating-route-controller-manager
+roleRef:
+  kind: Role
+  name: system:openshift:sa-creating-route-controller-manager
+subjects:
+- kind: User
+  namespace: openshift-route-controller-manager
+  name: system:route-controller-manager
```

Comment:
also `system:openshift:tokenreview-openshift-route-controller-manager` cluster role + binding should be present

Reply:
Done